### DISCUSSION TITLE
部署在 Vercel 上的性能表现

### DISCUSSION THREAD

**Discussion by acotor** [lobehub/lobehub#1788] [General Question]
fork 本项目后部署到 vercel，Speed Insights 页面显示性能存在不足。实际使用体验上也有明显感知。例如从会话页面切换至设置页面，页面的切换不跟手，延迟大概2秒左右；例如在会话页面切换不同的会话时，也存在类似的不跟手、延迟现象。

请问有没有基于 vercel 的优化手段？

![](https://s2.loli.net/2024/03/28/3xH2EKeqLBnQsAC.png)

**Comment by cy948** [comment]
layout重构 #1700
